common/predictive-text/worker/model-compositor.ts

```diff
@@ -410,7 +410,10 @@ class ModelCompositor {
     // Since we're outside of the standard `predict` control path, we'll need to
     // set the Reversion's ID directly.
     let reversion = this.toAnnotatedSuggestion(firstConversion, 'revert');
-    if(suggestion.id) {
+    if(suggestion.transformId || suggestion.transformId === 0) {
```

Comment:
Is this supposed to test that `transformId` is a number? Might be better to test that directly with `!== null` or similar?

Reply:
Or even `!= null` [which captures `undefined` as well](https://gist.github.com/eddieantonio/2664cc5902516035949b732bbe19ac2b).

Reply:
Well, more that `transformId` is defined and initialized.  Though... yeah, I guess it is about the same thing.  :+1: